src/types/ScreenInfoUiaProviderBase.cpp

```diff
@@ -32,10 +32,11 @@ SAFEARRAY* BuildIntSafeArray(std::basic_string_view<int> data)
 }
 
 #pragma warning(suppress : 26434) // WRL RuntimeClassInitialize base is a no-op and we need this for MakeAndInitialize
-HRESULT ScreenInfoUiaProviderBase::RuntimeClassInitialize(_In_ IUiaData* pData) noexcept
+HRESULT ScreenInfoUiaProviderBase::RuntimeClassInitialize(_In_ IUiaData* pData, _In_ std::wstring wordDelimiters) noexcept

```

Comment:
oh man you missed one
```suggestion
HRESULT ScreenInfoUiaProviderBase::RuntimeClassInitialize(_In_ IUiaData* pData, _In_ std::wstring& wordDelimiters) noexcept
```